istio-control/istio-autoinject/values.yaml

```diff
@@ -59,6 +59,12 @@ sidecarInjectorWebhook:
   neverInjectSelector: []
   alwaysInjectSelector: []
 
+  # Enable objectSelector to filter out pods with no need for sidecar before calling istio-sidecar-injector.
+  # It is disabled by default since this function will only work after k8s v1.15.
+  objectSelector:
+    enabled: false
+    autoInject: true
```

Comment:
https://istio.io/docs/setup/additional-setup/sidecar-injection/#more-control-adding-exceptions needs to be updated to reflect this new option.

NamespaceSelector → ObjectSelector → Pod Annotations → NeverInjectSelector → AlwaysInjectSelector → DefaultPolicy